src/main/java/org/stellar/anchor/paymentsservice/Balance.java

```diff
@@ -5,8 +5,20 @@
 @Data
 public class Balance {
     String amount;
+    /**
+     * The name of the currency that will be ultimately credited into the beneficiary user account. It should obey the
+     * {scheme}:{identifier} format described in <a href="https://stellar.org/protocol/sep-38#asset-identification-format">SEP-38</a>.
+     */
```

Comment:
❓ Are Java comments HTML?

Reply:
They support HTML to some extent. I've only used it for external links `<a href=...</a>` and for [code snippets](https://github.com/stellar/java-stellar-anchor-dev/blob/1a7268657a335397961629ac19e1dee73e575301/src/main/java/org/stellar/anchor/paymentsservice/PaymentsService.java#L99-L113).

In [this](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) blog post there are some examples of HTML being used to organize the text with paragraphs `<p>` and bullet points `<li>` as well.